BrickKit/bricks/src/main/java/com/wayfair/brickkit/brick/BaseBrick.java

```diff
@@ -20,6 +20,7 @@
  * Abstract class which defines Bricks.
  */
 public abstract class BaseBrick {
+    public static final int DEFAULT_LAYOUT_RES_ID = 0;
```

Comment:
is this even needed any more if this will never happen?

Reply:
It does/can happen.  It's just a constant to get away from using hardcoded zeros in multiple locations.